### PR TITLE
MAPS annotations

### PR DESCRIPTION
**Pull request by mumichae**
Provide different outputs for MAPS annotations

- removed `filter` wildcard
- remove `convert_bed` rule & script
- new wildcard for annotations
- new function to annotate variants by intervals
- moved gnomAD filtering an annotation functions to separate module `utr3variants`
- updated plots

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# pylint: disable=missing-module-docstring
+from setuptools import setup
+
+setup(
```
Comment (vladsavelyev) [inline setup.py:4]: Did you mean to fill the empty fields later?

**Review by vladsavelyev — APPROVED** [review]
Great job! 

Not going to dive into hail-heavy functions like `maps` as I trust there are adapted from `gnomad_lof`. Otherwise looks good to me.

File: setup.py (hunk 1 above)
Comment (mumichae) [reply setup.py:4]: Yes, depending on whether we are interested in a package or not. For now I just needed a package to collect functions in.

**Comment by mumichae** [comment]
Thanks, I didn't modify the hail functions much, just moved them to a new location

File: setup.py (hunk 1 above)
Comment (mumichae) [reply setup.py:4]: Yes, good point. It depends on whether we actually want a standalone package. For now I'm just using it to collect functions I'm using. Will leave it as is for now.

**Pull request merged**